bulk-updater/src/operations.rs

```diff
@@ -0,0 +1,304 @@
+        let out = osc_command(&pkgpath, &arguments)?;
+        if out.status.success() {
+            let command_output = String::from_utf8_lossy(&out.stdout);
+            tracing::info!("✅ -- osc clean operation passed");
+            tracing::info!("stdout -- {}", command_output);
+        } else {
+            let command_output = String::from_utf8_lossy(&out.stderr);
+            tracing::info!("🚨 -- osc clean operation failed");
+            tracing::info!("stderr -- {}", command_output);
+        };
+        let arguments = vec!["update", "."];
+        let out = osc_command(&pkgpath, &arguments)?;
+        if out.status.success() {
+            let command_output = String::from_utf8_lossy(&out.stdout);
+            tracing::info!("✅ -- osc update operation passed");
+            tracing::info!("stdout -- {}", command_output);
+        } else {
+            let command_output = String::from_utf8_lossy(&out.stderr);
+            tracing::info!("🚨 -- osc update operation failed");
+            tracing::info!("stderr -- {}", command_output);
+        };
+    } else {
+        let arguments = vec!["bco", "."];
+        let out = osc_command(&pkgpath, &arguments)?;
+        if out.status.success() {
+            let command_output = String::from_utf8_lossy(&out.stdout);
+            tracing::info!("✅ -- osc bco operation passed");
+            tracing::info!("stdout -- {}", command_output);
+        } else {
+            let command_output = String::from_utf8_lossy(&out.stderr);
+            tracing::info!("🚨 -- osc bco operation failed");
+            tracing::info!("stderr -- {}", command_output);
+        };
+    }
+    Ok(pkgpath)
+}
+
+pub fn attempt_submit(pkgpath: &Path, message: &str, yolo: bool) -> io::Result<PathBuf> {
+    let pkgpath_str = pkgpath.to_string_lossy();
+    if yolo {
+        let arguments = vec!["vc", "-m", message];
+        let out = osc_command(pkgpath, &arguments)?;
+        if out.status.success() {
+            let command_output = String::from_utf8_lossy(&out.stdout);
+            tracing::info!("✅ -- osc vc operation passed");
+            tracing::info!("stdout -- {}", command_output);
+        } else {
+            let command_output = String::from_utf8_lossy(&out.stderr);
+            tracing::info!("🚨 -- osc vc operation failed");
+            tracing::info!("stderr -- {}", command_output);
+        };
+        let arguments = vec!["ci", "-m", message];
+        let out = osc_command(pkgpath, &arguments)?;
+        if out.status.success() {
+            let command_output = String::from_utf8_lossy(&out.stdout);
+            tracing::info!("✅ -- osc ci operation passed");
+            tracing::info!("stdout -- {}", command_output);
+        } else {
+            let command_output = String::from_utf8_lossy(&out.stderr);
+            tracing::info!("🚨 -- osc ci operation failed");
+            tracing::info!("stderr -- {}", command_output);
+        };
+        let arguments = vec!["sr", "-m", message];
```

Comment:
maybe we need an extra option for "opts.findout" which does the submit ontop of yolo. That way we can do a yolo commit, but then manually sr, or we can "findout" and just go ham.

Reply:
so findout is a param if we want to manually sr or not?